src/TemplateHelpers.hs

```diff
@@ -125,7 +128,18 @@ renderReadme = \case
 renderHtmlDocs :: D.VerifiedPackage -> Text -> Handler (Maybe Html)
 renderHtmlDocs pkg mnString = do
   htmlRenderContext <- getHtmlRenderContext
-  let docsOutput = packageAsHtml (htmlRenderContext pkg) pkg
+
+  depHtmlRenderContexts <- do
```

Comment:
I found this a little difficult to follow, keeping track of how many arguments have been applied to the `htmlRenderContext` function at a given point. How about extracting a function `collectDepsHtmlRenderContexts :: Package a -> Handler (Bower.PackageName -> P.ModuleName -> Maybe D.HtmlRenderContext)`?